lib/util/cache/repository/impl/repository-cache-null-impl.ts

```diff
@@ -1,6 +1,6 @@
 import type { RepoCache, RepoCacheData } from '../types';
 
-export class RepoCacheBase implements RepoCache {
+export class RepositoryCacheNullImpl implements RepoCache {
```

Comment:
```suggestion
export class NullRepositoryCache implements RepoCache {
```

remove `Impl` suffix from all